manifests/t/Trufflesuite/Ganache/Beta/2.6.0-beta.3/Trufflesuite.Ganache.Beta.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Trufflesuite.Ganache.Beta
+PackageVersion: 2.6.0-beta.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
```

```suggestion
  Scope: user
```
If it doesn't install in `%LOCALAPPDATA%` or `%USERPROFILE%`, it's `Scope: machine`, otherwise, it's `Scope: user`.